tests/e2e/k8s/util/k8s_yaml.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2023 The ClusterLink Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/clusterlink-net/clusterlink/pkg/bootstrap/platform"
+)
+
```

Comment:
Nope, they are specific to the test usage.

Reply:
But if this is something that you want to change, instead of doing a search and replace, a better option is to add it to cl-adm, probably also the user will probably want to do it, like define namespace or create nodeport/load-balancer/clusterip service

Reply:
I agree we can extend cl-adm to support various parameters.
It may be in the "new" cl-adm, which will be operator-based instead of CLI.
I think this is currently out of the scope of this PR.